packages/orchestration/src/chain-capabilities.js

```diff
@@ -38,23 +38,43 @@ const PfmEnabled = /** @type {const} */ ({
 });
 harden(PfmEnabled);
 
+const IcaEnabled = /** @type {const} */ ({
```

Comment:
from where is this info derived? what's the process to update it?

Reply:
Manually derived. To update automatically we might be able to get a public RPC endpoint for each chain from the chain registry, then query the endpoints for the ICA info. However, the RPC endpoints in chain registry are often unreliable.